dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpChannelConfig.java

```diff
@@ -1226,113 +1248,110 @@ private void parseAccessLog(Map<Object, Object> props) {
                 Tr.event(tc, "Http Channel Config: SameSite Lax configuration parsed.");
             }
         }
 
     }
 
-    private void parseCookiesSameSiteNone(Map<Object, Object> props) {
-        Object value = props.get(HttpConfigConstants.PROPNAME_SAMESITE_NONE);
-        if (null != value && this.useSameSiteConfig) {
-
-            if (value instanceof String[]) {
-                String[] cookies = (String[]) value;
+    protected void parseCookiesSameSiteNone(Object option) {
+        if (Objects.nonNull(option) && this.useSameSiteOptions) {
+            if (option instanceof String[]) {
+                String[] cookies = (String[]) option;
                 for (String s : cookies) {
-
                     addSameSiteAttribute(s, HttpConfigConstants.SameSite.NONE);
-
                 }
-            }
-            if (this.useSameSiteConfig && (TraceComponent.isAnyTracingEnabled()) && (tc.isEventEnabled())) {
-                Tr.event(tc, "Http Channel Config: SameSite None configuration parsed.");
+                if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
+                    Tr.event(tc, "Http Channel Config: SameSite None configuration parsed.");
+                }
             }
         }
     }
 
-    private void parseCookiesSameSiteStrict(Map<Object, Object> props) {
-        Object value = props.get(HttpConfigConstants.PROPNAME_SAMESITE_STRICT);
-        if (null != value && this.useSameSiteConfig) {
-
-            if (value instanceof String[]) {
-                String[] cookies = (String[]) value;
+    protected void parseCookiesSameSiteStrict(Object option) {
+        if (Objects.nonNull(option) && this.useSameSiteOptions) {
+            if (option instanceof String[]) {
+                String[] cookies = (String[]) option;
                 for (String s : cookies) {
-
                     addSameSiteAttribute(s, HttpConfigConstants.SameSite.STRICT);
+                }
+                if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
+                    Tr.event(tc, "Http Channel Config: SameSite Strict configuration parsed.");
+                }
+            }
+        }
+    }
+
+    protected void parseCookiesSameSitePartitioned(Object option) {
+        if (Objects.nonNull(option) && this.useSameSiteOptions) {
+            if (option instanceof Boolean) {
 
+                Boolean partitionedValue = (Boolean) option;
+                if (partitionedValue) {
+                    this.isPartitioned = true;
                 }
             }
-            if (this.useSameSiteConfig && (TraceComponent.isAnyTracingEnabled()) && (tc.isEventEnabled())) {
-                Tr.event(tc, "Http Channel Config: SameSite Strict configuration parsed.");
+            if (this.useSameSiteConfig() && (TraceComponent.isAnyTracingEnabled()) && (tc.isEventEnabled())) {
+                Tr.event(tc, "Http Channel Config: SameSite Partitioned configuration parsed.");
```

Comment:
We changed this log message from strict to Partitioned. Is this something we should be changing? Or something we should change separately?

Reply:
https://github.com/OpenLiberty/open-liberty/pull/32482/